meetings/2022-11-07.md

```diff
@@ -0,0 +1,107 @@
+* AC: I think I have a good idea about the problem.
+
+ACTION: AC to create issue tracking disjont definition of leaf entity and trust 
+
+* ES: I understand that federation works with the same claims as dynamic registration. Is there a way to add more data? Let's say we have trus anchor at solidproject.org, can it say that all clients registered with it have a flag that it marks it as 'in preview' or untrusted. It would be helpful with any app marketplace to mark apps.
+* AC: Yes you can add metadata. Inside of Entity Configuration there is *metadata* secion. It is generally what we would see in .well-known/openid-configuration for the RP there would be mostly dynamic client registration information. There is also *metadata policy* which can for example define constraints in all entities in the federation, which is kind of nice. It can set high level constraints in the federation which can be well defined in the whole system.
+* ES: People often mix localhost and production URLs together, which I recommend since it says it is the same client. To be able to have a federation server, which says *all documents registered with me are developmen preview*. On the consent pages we can flag that aspec of the application. I think trust will be a big part in the future for any application.
+* AC: I hear practically from all of us that trust is important. What OIDC Federation can give us provides much better foundation for building this trust. We can devise a specific federation which is for development. When you use that app with your pod you will know how much you can trust it. Now everything is a hodge podge.
+* AC: I would like a strawman vote just to gauge opinions.
+* eP: In support of exploring this. Am interested in drafting a PR to the primer, where we can see what form this will take. That will give us an idea for how the spec change would look.
+* ES: Getting the primer written first will be a good step. Showing how the things will work in context of solid will help.
+* WT: I think this is a good idea, we should look for minimal setup based on OIDC Federation.
+
+ACTION: AC to add note to the issue 207 that we were all in favor of pursuing it and fleashing out all the details.
+
+* ES: Would the OP for Solid Server would be the same as provider of the federation server? For example would one go to id.inrupt.net to register the client.
+* AC: I feel like one could have a federation which would be tied to OP but it could also be independent. If I were to write it I would probably keep them sepeate. One can have metadata for RP but also for OPs. If you want to be a root for OPs you would want to be independent from a specific OP.
+* WT: It is really attractive to keep them separate but also bundling multiple providers in ecosystem. The federation can put extra requrements for security and once OP complies it could be added to the federation.
```

Comment:
```suggestion
* WT: It is really attractive to keep them separate but also bundling multiple providers in ecosystem. The federation can put extra requirements for security, and once OP complies, it could be added to the federation.
```